### ISSUE TITLE
WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (input[aria-label="Escreva um destino"])

### ISSUE THREAD

**Issue by Erikapp** [Erikapp/accessibility-test#9]
#### Issue
`Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Aeroméxico](https://www.aeromexico.com/pt-br)
#### Element path
input[aria-label="Escreva um destino"]
#### Snippet
`<input type="text" aria-label="Escreva um destino" autocomplete="off" class="FlightNewbooker-input" name="destination" placeholder="Escreva um destino" value="">`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.81 (foreground color: #8d807a, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 137.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
